frontend/src/hooks/useFunctions.tsx

```diff
@@ -0,0 +1,20 @@
+import type { FunctionType } from '@/pages/dashboard';
+import { AxiosPrivate } from '@/utils';
+import { useEffect, useState } from 'react';
+
+const useFunctions = () => {
+  const [functions, setFunctions] = useState<FunctionType[]>([]);
+
+  useEffect(() => {
+    (async () => {
+      const { data } = await AxiosPrivate.get('/function');
+      setFunctions(data);
```

Comment:
Could we get this from the useGetFilters hook? (if prefer alone this is good too!)

Reply:
No